### PR TITLE
Adding Linux and Windows build Jobs

### PR DESCRIPTION
**Pull request by Psypher9**
- Adding export presets
- Adding windows and linux package build jobs; #165 Also added bug emjoi to the bug template

### CODE REVIEW

File: poofarmer/project.godot [hunk 1]
```diff
@@ -29,7 +29,7 @@ _global_script_classes=[ {
 "language": "GDScript",
 "path": "res://Scripts/Gnome.gd"
 }, {
-"base": "Enemy",
+"base": "KinematicBody2D",
```
Comment (EthanHair) [inline poofarmer/project.godot:32]: Was this intentional?

File: poofarmer/project.godot [hunk 2]
```diff
@@ -185,11 +185,6 @@ railgun={
 "events": [ Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"alt":false,"shift":false,"control":false,"meta":false,"command":false,"pressed":false,"scancode":54,"physical_scancode":0,"unicode":0,"echo":false,"script":null)
  ]
 }
-gobodar={
-"deadzone": 0.5,
-"events": [ Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"alt":false,"shift":false,"control":false,"meta":false,"command":false,"pressed":false,"scancode":57,"physical_scancode":0,"unicode":0,"echo":false,"script":null)
- ]
-}
```
Comment (EthanHair) [inline poofarmer/project.godot:192]: And this?

**Review by EthanHair — CHANGES_REQUESTED** [review]

File: poofarmer/project.godot (hunk 1 above)
Comment (Psypher9) [reply poofarmer/project.godot:32]: No I think this might have been old stuff in my project.godot

Comment (Psypher9) [reply poofarmer/project.godot:32]: We _want _ this to be an `Enemy` right?

Comment (EthanHair) [reply poofarmer/project.godot:32]: Yes

File: poofarmer/project.godot (hunk 2 above)
Comment (EthanHair) [reply poofarmer/project.godot:192]: And this input should be there too

Comment (Psypher9) [reply poofarmer/project.godot:192]: Alright, I'll put these back. I don't know what edited these

File: poofarmer/project.godot (hunk 1 above)
Comment (Psypher9) [reply poofarmer/project.godot]: I see now, I at first thought this was setting types up in the Main scene, but I see that it was setting the types for the scenes in the whole project

File: poofarmer/project.godot (hunk 2 above)
Comment (Psypher9) [reply poofarmer/project.godot]: updated

**Review by EthanHair — APPROVED** [review]

**Pull request merged**